### DISCUSSION TITLE
Is there a way to show FPS when gaming?

### DISCUSSION THREAD

**Discussion by reinaldo-br** [mathoudebine/turing-smart-screen-python#193] [Q&A] [answered]
As the title says: I would like to see, among the info displayed, the FPS of the games I´m playing. Is there a way to do so?

**Comment by ooemir23** [comment]
@reinaldo-br  hi bro. You have a solitions this for quesiton?

**Comment by mathoudebine** [comment, marked as answer]
This has now been implemented to the latest 3.3.1 release.
Feel free to try it out and add the FPS framerate to your favorite theme!

Please note it is for now limited to Windows platforms and AMD GPUs

**Reply by aStroNuts04** [reply]
hi, ive heard for turzx screens there is a way to get it to work with afterburner and rivatuner for nvidia. Has there been a workaround for nvidia gpus?
